test/dynamo/test_graph_region_tracker.py

```diff
@@ -0,0 +1,251 @@
+# Owner(s): ["module: dynamo"]
+import torch
+import torch.fx
+from torch._dynamo.symbolic_convert import InstructionTranslator
+from torch._dynamo.test_case import TestCase
+from torch.utils._pytree import tree_map
+
+
+def extract_graph_and_tracker(fn, *args, **kwargs):
+    gm = None
+    region_tracker = None
+
+    def extract_graph_backend(_gm, *args, **kwargs):
+        nonlocal gm
+        nonlocal region_tracker
+        gm = _gm
+        region_tracker = InstructionTranslator.current_tx().output.region_tracker
+        return _gm
+
+    torch.compile(backend=extract_graph_backend)(fn)(*args, **kwargs)
```

Comment:
is this assuming fullgraph=True?

Reply:
Yeah I'll add that, my examples just happened to be full graphs :)